packages/commonwealth/client/scripts/views/components/ThreadContestTag/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { AssociatedContest } from 'models/Thread';
+import moment from 'moment/moment';
+
+export const getWinnersFromAssociatedContests = (
+  associatedContests?: AssociatedContest[],
+) => {
+  if (!associatedContests) {
+    return [];
+  }
+
+  return associatedContests
+    .map((contest) => {
+      const hasEnded = moment(contest.end_time) < moment();
+      const isActive = contest.contest_cancelled ? false : !hasEnded;
+
+      if (isActive) {
+        return null;
+      }
+
+      return {
+        date: moment(contest.end_time).format('DD/MM/YYYY'),
```

Comment:
can multiple contests happen in a single day? If yes, should we display the time there as well?

Reply:
I dont think you need to show the time here -  we want to encourage one contest per topic, and the above solution works for that